packages/fe-fpm-writer/src/filter/types.ts

```diff
@@ -0,0 +1,43 @@
+import type { CustomElement, InternalCustomElement, Position, EventHandler } from '../common/types';
+
+/**
+ * Properties to create a custom filter.
+ */
+export interface CustomFilter extends CustomElement, EventHandler {
+    /**
+     * Filter label in the application.
+     */
+    label: string;
+    /**
+     * Property used to filter by.
+     */
+    property: string;
+    /**
+     * ID of the embedded filter XML view.
+     */
+    controlID: string;
+    /**
+     * Existing filter XML fragment file name without '.fragment.xml' suffix.
+     * If not set, is equal to the name specified in the config.
+     */
+    fragmentFile?: string;
+    /**
+     * Name of the created filter function.
+     * If function name is not set in the eventHandler, is equal to 'itemsFilter'.
+     */
+    eventHandlerFnName?: string;
```

Comment:
Why do we have this on on the public interface again, shouldn't it be already possible to set it with `eventHandler.fnName`?